content/blog/2023-04-13-rse-on-mastodon.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "State of Research software engineers on Mastodon"
+
+[extra]
+
+author = "Richard Darst"
++++
+
+Many domains have migrated to Mastodon as of 2023.  Have Research
+Software Engineers?  This post examines the situation, based on [this
+post by rkdarst](https://fosstodon.org/@rkdarst/110146601969193973)
+
+Summary: not much yet
+
+This post can be updated as we get more information/the situation
+evolves, please send pull requests.
+
+
+## RSE organizations
+
+* US RSE: <https://fosstodon.org/@us_rse>
+* DE RSE: <https://fosstodon.org/@de_rse@mastodon.social>
+* Nordic RSE: <https://fosstodon.org/@nordic_rse>
+* Society of RSE (UK/int): ??
+* ... ?
+
+
+## Other accounts to follow to seed your community
+
+TODO (should we join some other list instead of make a new one here?)
+
+
+## Common hashtags
+
+* **#RSEng and #RSE** have been seen, but #RSEng is preferred
+  ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502) (same
+  as Twitter).  There doesn't seem to be much activity there.
```

Comment:
```suggestion
  as Twitter).  There doesn't seem to be much activity there.  #RSEng was
  the primary tag from from Twitter.  Sometimes #RSE is the people and
  #RSEng is the activity.
```

Reply:
“(same as Twitter)” not needed given next sentence explains further.